### ISSUE TITLE
[A11Y] Missing focus styling for URL field

### ISSUE THREAD

**Issue by softvision-oana-arbuzov** [webcompat/webcompat.com#3423]
**Browser / Version**: Firefox Nightly 81.0a1 (2020-07-27), Chrome 84.0.4147.89 
**Operating System**: Windows 10 Pro

**Steps to Reproduce**:
1. Navigate to https://webcompat.com/issues/new
2. Type in the URL and select the text.
3. Observe styling when URL is in focus.

**Expected Behavior:**
There is styling (different text color/background/boarder color).

**Actual Behavior:**
There is no styling.

**Notes:**
1. Screenshot attached.
2. According to "Inspector>Accessibility" tab, "Focusable element must have focus styling". 
![image](https://user-images.githubusercontent.com/12184325/88675174-55014b80-d0f3-11ea-9afe-e85d59571eb8.png)
Read more here: https://developer.mozilla.org/en-US/docs/Web/CSS/:focus
3. Also there is no "keyboardShortcut:". 

**Watchers:**
@softvision-oana-arbuzov
@cipriansv

**Comment by miketaylr** [comment]
We do have focus styling -- if you focus the input, we have a `:focus` style here:

```
.issue-form .text-field:focus {
 box-shadow:0 0 0 3px #56638c;
 box-shadow:0 0 0 3px var(--issue-type-bg)
}
```

Possibly the a11y inspector is just looking for a border or outline?

This is implemented here: https://searchfox.org/mozilla-central/source/devtools/server/actors/accessibility/audit/keyboard.js#167-217

It's clever -- it collects all relevant styles for an element, then locks in the `:focus` style for that element (if any), and if they are the same as the un-focused styles, it claims there are no focus styles.

So what's up with what we're doing...

Ah, Ok. I see the issue.

@softvision-oana-arbuzov did the test while the URL field is in the error state, which applies this rule:

```
.issue-form .is-error .text-field {
    border-color: #e22850;
    border-color: var(--wizard-step-error);
}
```

We don't have an error-specific `:focus` state, so the tool things we're doing something wrong. If you re-run the test on the input while not in an error-state it doesn't claim we don't have focus styles:

![Screen Shot 2020-08-06 at 4 46 10 PM](https://user-images.githubusercontent.com/67283/89586119-b49edb80-d804-11ea-9d81-d0d2b4d2e286.png)

**Comment by miketaylr** [comment]
I think we can close this as invalid, but if we think we should have keyboard shortcut buttons defined, we should file a different bug for that. We'll need to make sure all steps in the form can be navigated by keyboard shortcuts which is more complex than just adding an `accesskey`: https://webaim.org/techniques/keyboard/accesskey#spec

**Comment by softvision-oana-arbuzov** [comment]
Thanks @miketaylr , I was not 100% sure what the exact issue was here, but I passed it on as much as I could understand. 

In regard to `keyboard shortcuts` I will look into it and find all the elements that are missing a shortcut key and report a new issue.

**Comment by miketaylr** [comment]
All good -- keep the reports coming!

**Comment by miketaylr** [comment]
(also, it was cool for me to learn more about how the a11y inspector tool works, so thanks for that)
